udp-server/setup.py

```diff
@@ -94,7 +96,11 @@ def prompt_player_tags(self):
 		"""
         for i in range(1, self.amount_of_players + 1):
             while True:
-                player_tag = input("Player {}'s tag: ".format(i))
+                if i < self.amount_of_players // 2 + 1:
```

Comment:
Er denne ikke altid true udover sidste iteration? Er det ikke intentionen at halvdelen bliver promptet for det ende hold og den anden halvdel det andet?

Reply:
nevermind det lignede at // 2 + 1 var en kommentar så nu giver det mening